game/game/playercontrol.cpp

```diff
@@ -430,6 +430,7 @@ void PlayerControl::marvinF8(uint64_t dt) {
   pl.clearSpeed();
   pl.quitIneraction();
   pl.setAnim(AnimationSolver::Idle);
+  pl.clearAiQueue();
```

Comment:
Is it intentional change? Commit message implies that it is not

Reply:
F8 stops `aigoto`  and all further queued `aigoto` commands are ignored.